dataset_zoo/sroie/textrecog.py

```diff
@@ -0,0 +1,77 @@
+data_root = 'data/sroie'
```

Comment:
It seems this config is very similar to `textdet.py`, so we can take advantage of config inheritance like what you did in `textspotting.py`, or [totaltext's recog confg](https://github.com/open-mmlab/mmocr/blob/dev-1.x/dataset_zoo/totaltext/textrecog.py)